hoprd/hoprd/example_cfg.yaml

```diff
@@ -84,7 +84,7 @@ hopr:
         # Number of acknowledged winning tickets in a channel that triggers the ticket aggregation
         # in that channel when exceeded.
         # This condition is independent of `unrealized_balance_ratio`.
-        aggregation_threshold: 1000
+        aggregation_threshold: 100
```

Comment:
Why so low? Under high load, e.g. 1000 packets per second, the aggregation is triggered quite frequently.

Reply:
That's not a problem though. Lots of smaller chunks are better than larger chunks, because they have lower chance of timing out.